src/main/java/NintendoSwitchPage.java

```diff
@@ -0,0 +1,17 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class NintendoSwitchPage {
+
+    WebDriver driver;
+
+    public NintendoSwitchPage(WebDriver driver){ this.driver = driver;}
+
+    public ArticleSwitchPage OpenNintendo(){
+        By NomLabel = By.cssSelector("[data-asin='B07WKNQ8JT']");
```

Comment:
use another selector, not the `data-asin` and put the selector outside of the function